### DISCUSSION TITLE
Display the human labeled mask on the img results

### DISCUSSION THREAD

**Discussion by kankanzheli** [open-mmlab/mmdetection#9119] [Q&A] [answered]
Sorry to ask this question. This question is relevant to my research. The current code can display the mask results of each object on the output image when running mask-RCNN. But I need to display the human marked mask results on the output image at the same time (I will do other operations later). Is there any way to help the current code achieve this?

As I know, you use the following code to show the mask results in the picture.
result = inference_detector(model, image_np)
bbox_result, segm_result = result
segms = mmcv.concat_list(segm_result)

My idea is to convert the mask result I marked into a secretary in the same format as the segms obtained in the above code. Is there any way to do this？
Please. This is very important for my research

In my opinion:

Because there are steps to calculate the mask IOU in the running process of your mask RCNN. In this step, you need to convert the annotation data from the coco dataset to the same format as your mask RCNN run results

**Comment by ZwwWayne** [comment, marked as answer]
You can use browse_dataset.py to visualize the ground truth of the dataset. You can also use imshow_gt_det_bboxes from https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/visualization/image.py#L380
